src/olympia/addons/tests/test_views.py

```diff
@@ -62,8 +63,9 @@ def check_cat_sidebar(url, addon):
     """Ensures that the sidebar shows the categories for the correct type."""
     for type_ in [amo.ADDON_EXTENSION, amo.ADDON_THEME, amo.ADDON_SEARCH]:
         addon.update(type=type_)
-        r = Client().get(url)
-        assert pq(r.content)('#side-nav').attr('data-addontype') == str(type_)
+        response = Client().get(url)
+        assert pq(response.content)('#side-nav').attr('data-addontype') == str(
```

Comment:
change to `text_type`

Reply:
Fixed in 71d47d41e01d219aedf5bd6e2bf74c21ec370019 with a few more str/bytes fixes that will matter latter for the `es` job.